### DISCUSSION TITLE
v.0.1.0-beta

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#89] [Releases]
## 🚀 Woohoo the 1st Beta Release!
Finally its time! With the base image out of beta and the introduction of database notifications and absolute thresholds it's time to stick a fork in `v0.1.0` and start moving on.

## What's Changed
* Added database notifications to filament by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/83
* Remove beta tag from Docker image 🎉 by @jaydrogers in https://github.com/alexjustesen/speedtest-tracker/pull/87
* Performance thresholds by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/85
* Disable database notifications by default by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/88

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.1.0-alpha.10...v.0.1.0-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v.0.1.0-beta'>v.0.1.0-beta</a>.</em>

**Comment by alexjustesen** [comment]
For those following along, I'm going to leave `v0.1.0-beta` out there for a week. If no issues are reported I'll tag `v0.1.0` and move on to the next major feature "pretty charts".

**Comment by alexdelprete** [comment]
there's a dot after the v of the semver tag. Couldn't pull the image and then I noticed it. :)

**Comment by alexdelprete** [comment]
Thresholds: really cool feature. :)

They're working, but in the setting panel I would specify it's a lower threshold: any test value BELOW those absolute settings will trigger the notification.

![image](https://user-images.githubusercontent.com/7027842/197923579-39c46a85-2b3b-4308-8611-425ef59d9b53.png)
